src/module/actor/data-model-classes/data-model-character-encumbrance.ts

```diff
@@ -17,8 +17,8 @@ export interface CharacterEncumbrance {
  export default class OseDataModelCharacterEncumbrance implements CharacterEncumbrance {
   static baseEncumbranceCap = 1600;
   static encumbranceSteps = {
```

Comment:
Eventually, I'd like to revisit the naming of this thing to make it more sensible for slot-based (or similar) encumbrance schemes.